src/test/java/org/stanford/ncbo/owlapi/wrapper/OntologyParserTest.java

```diff
@@ -125,6 +126,25 @@ public void parse_OntologyWithEmbeddedXML_ReturnsTrue() throws Exception {
         assertTrue(parser.parse());
     }
 
+
```

Comment:
I think the new unit test could be simplified in a way that eliminates the need for having added the `getTargetOwlOntology` method to the `OntologyParser` class.

You could do something like the following instead:

```java
ParserInvocation pi = new ParserInvocation("./src/test/resources/repo/input/cno", "./src/test/resources/repo/output/cno", "cnov0_5.owl", true);
OntologyParser parser = new OntologyParser(pi);
assertTrue(parser.parse());

IRI iri = IRI.create("http://purl.org/incf/ontology/Computational_Neurosciences/cno_alpha.owl");
Optional<IRI> parsedIRI = parser.getParsedOntologies().stream().findFirst().get().getOntologyID().getOntologyIRI();
assertEquals(iri, parsedIRI.get());
```

Reply:
I don't think this will work.

My goal was **to test if the generated ontology (owlapi.xrdf) contains an ontology IRI**.

The generated ontology can be found only in the `targetOwlOntology`, while the `parser.getParsedOntologies().stream().findFirst().get().getOntologyID().getOntologyIRI()` will return the source ontology.